### DISCUSSION TITLE
Not seeing any confirmations

### DISCUSSION THREAD

**Discussion by Gebster28** [Ride-The-Lightning/RTL#1346] [Troubleshooting Questions]
Hi I made a payment onchain, and the transaction was mined and I see it in the mempool. But how come on LND I do not see the confirmations it shows zero?

<img width="670" alt="TX1" src="https://github.com/Ride-The-Lightning/RTL/assets/119563706/854386f0-da94-477c-95ac-2f97ecef154e">

**Comment by saubyk** [comment]
Hmm...I am also wondering why the amount is displayed as -ve
Would it possible for you to share the transaction hash?

**Reply by Gebster28** [reply]
sure, here it is:
7c137b293050d09c51c78a84b672e48c28e5a7cba1a06dafc62b2fe3d832e29a

**Reply by saubyk** [reply]
Couple of more questions:
- Is your node synced to the chain?
- Which version of LND are you on?
- Did you try to restart the node?

**Reply by Gebster28** [reply]
No, the node seems to be still synching
LND 0.7.1.3
I will try and reboot the node

**Reply by saubyk** [reply]
You must have the node synced to the chain, to be able to get the accurate status of transactions from it

**Reply by Gebster28** [reply]
seems to be an issue with synching over the last few days. finally I see the status is no accurate. thank you
